modules/admin-ui-picard/app/src/utils/resourceUtils.js

```diff
@@ -98,6 +105,50 @@ export const prepareMetadataFieldsForPost = (metadataInfo, values) => {
     return metadataFields;
 }
 
+export const prepareSeriesMetadataFieldsForPost = (metadataInfo, values) => {
+    let metadataFields = [];
+
+    // fill metadataField with field information send by server previously and values provided by user
```

Comment:
grammar:  it should be `sent by server previously`, not `send by server previously`